api/server/structs/endpoints_beacon.go

```diff
@@ -169,7 +169,8 @@ type BLSToExecutionChangesPoolResponse struct {
 }
 
 type GetAttesterSlashingsResponse struct {
-	Data []*AttesterSlashing `json:"data"`
+	Version string          `json:"version"`
```

Comment:
In https://github.com/prysmaticlabs/prysm/pull/14481/files#diff-b7de0123b995294eb1e799afb080f62c9a4a08195f9ed1f387a9071922b1ec97 the `Version` field has the `omitifempty` tag. I believe this is required for v1 to work correctly?

Reply:
It does seem to work correctly but for the sake of accuracy we should definitely add the field